region/crateria/east/West Ocean.json

```diff
@@ -2979,13 +2938,13 @@
     },
     {
       "id": 134,
-      "link": [17, 10],
-      "name": "G-mode Overload PLMs by PBing Missile Item",
+      "link": [17, 1],
+      "name": "G-mode Overload PLMs by Power Bombing Morph Maze Item",
       "requires": [
-        {"notable": "G-mode Overload PLMs by PBing Missile Item"},
+        {"notable": "G-mode Overload PLMs by Power Bombing Morph Maze Item"},
         "canEnterGMode",
-        {"itemNotCollectedAtNode": 10},
         "h_canUsePowerBombs",
+        "canInsaneJump",
```

Comment:
This wasn't a hard jump, from when I had to do this last.

Reply:
It was mostly used for the obscurity and precision of the PB, and if someone wanted to play with just an easier set of g-mode strats, or if the easier strats were moved down into Expert. But I think you're right, that this isnt the way to do it. I'll remove it and check if there were any others that were added like this in Crateria and remove them and think more on how it should be addressed.